CMakeLists.txt

```diff
@@ -1,4 +1,12 @@
 cmake_minimum_required(VERSION 3.13.0)
+
+if(WIN32 AND NOT CMAKE_TOOLCHAIN_FILE)
+  set(CMAKE_TOOLCHAIN_FILE "${CMAKE_PREFIX_PATH}/scripts/buildsystems/vcpkg.cmake"
+    CACHE STRING "vcpkg toolchain file")
+  set(VCPKG_TARGET_TRIPLET "x64-windows")
+  set(X_VCPKG_APPLOCAL_DEPS_INSTALL ON)
+endif()
```

Comment:
Unsure about this. One one hand this doesn't belong into our project CMakeLists.txt IMHO, on the other hand it makes it easier to build from the command line on windows (for visual studio this is irrelevant because we can set these cars in CMakeSettings.json).

Reply:
vcpkg recommends it:
https://github.com/microsoft/vcpkg/#vcpkg-as-a-submodule

Reply:
Only when you use it as a submodule though. Acquiring dependencies is in the user's domain IMHO, but these lines make it impossible to build without a toolchain file.